### PR TITLE
Updated to support v5

### PR DESCRIPTION
**Pull request by TheIncorrigible1**
I also updated the style / logic to be cleaner / more consistent.

### CODE REVIEW

**Comment by TheIncorrigible1** [comment]
Should address #3

**Comment by kiedtl** [comment]
Everything looks great - but why did you remove the `-help` flag? I think we should have both the cmdlet docs (sorry, not sure what it's called) and a help flag.

Otherwise, kudos!

File: src/winfetch.ps1 [hunk 1]
```diff
@@ -24,470 +22,420 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-# Summary: Winfetch is a command-line system information utility for Windows written in PowerShell.
```
Comment (kiedtl) [inline src/winfetch.ps1]: May I ask why the help section is removed?

Comment (TheIncorrigible1) [reply src/winfetch.ps1]: I changed it to use the standard comment-based help system so you could do `Get-Help filename.ps1`  You could also apply a URL to this and add `-Online` to get directed to the github readme.

Comment (kiedtl) [reply src/winfetch.ps1]: That's good. But lets still keep the help flag. :)

**Pull request merged**